--- .bazelrc	2024-05-20
+++ .bazelrc	2024-05-20
@@ -165,6 +165,7 @@
 build:windows --action_env=CXX_x86_64_pc_windows_gnu=/usr/bin/x86_64-w64-mingw32-g++
 build:windows --action_env=AR_x86_64_pc_windows_gnu=/usr/bin/x86_64-w64-mingw32-ar
 build:windows --action_env=LD_x86_64_pc_windows_gnu=/usr/bin/x86_64-w64-mingw32-ld
+build:windows --action_env=DLLTOOL_x86_64_pc_windows_gnu=/usr/bin/x86_64-w64-mingw32-dlltool
 # Workaround to fix bazel analysis failure on Windows CI
 build:windows --build_tag_filters=-requires_x11,-requires_wayland,-e2e
