--- src/app/Cargo.toml
+++ src/app/Cargo.toml
@@ -41,4 +41,3 @@ slint-build = "1.8.0"
 tonic-build = "0.12"

 [dev-dependencies]
-mockito = "1.4.0"
--- src/app/BUILD.bazel
+++ src/app/BUILD.bazel
@@ -204,7 +204,6 @@ rust_test(
         "@crates//:chrono",
         "@crates//:copypasta",
         "@crates//:futures",
-        "@crates//:mockito",
         "@crates//:prost",
         "@crates//:reqwest",
         "@crates//:serde",
