--- srcs/server/telemetry/BUILD.bazel	2024-05-08 02:57:00.000000000 +0000
+++ srcs/server/telemetry/BUILD.bazel	2024-05-08 03:42:00.000000000 +0000
@@ -6,6 +6,7 @@
         "harness_metrics.go",
         "local_buffer.go",
         "rag_sync_metrics.go",
+        "pii.go",
     ],
     importpath = "onehumancorp/srcs/server/telemetry",
     visibility = ["//visibility:public"],
@@ -19,6 +20,7 @@
     name = "telemetry_test",
     srcs = [
         "harness_metrics_test.go",
         "local_buffer_test.go",
+        "pii_test.go",
     ],
     embed = [":telemetry"],
