diff --git a/rust/platform/triple_mappings.bzl b/rust/platform/triple_mappings.bzl
index 06d27a1..fbad45d 100644
--- a/rust/platform/triple_mappings.bzl
+++ b/rust/platform/triple_mappings.bzl
@@ -49,6 +49,7 @@ SUPPORTED_T2_PLATFORM_TRIPLES = {
     "aarch64-apple-ios-macabi": _support(std = True, host_tools = False),
     "aarch64-apple-ios-sim": _support(std = True, host_tools = False),
     "aarch64-linux-android": _support(std = True, host_tools = False),
+    "aarch64-pc-windows-gnullvm": _support(std = True, host_tools = False),
     "aarch64-pc-windows-msvc": _support(std = True, host_tools = True),
     "aarch64-unknown-fuchsia": _support(std = True, host_tools = False),
     "aarch64-unknown-uefi": _support(std = True, host_tools = False),
@@ -75,6 +76,7 @@ SUPPORTED_T2_PLATFORM_TRIPLES = {
     "wasm32-wasip1-threads": _support(std = True, host_tools = False),
     "wasm32-wasip2": _support(std = True, host_tools = False),
     "x86_64-apple-ios": _support(std = True, host_tools = False),
+    "x86_64-pc-windows-gnullvm": _support(std = True, host_tools = False),
     "x86_64-apple-ios-macabi": _support(std = True, host_tools = False),
     "x86_64-linux-android": _support(std = True, host_tools = False),
     "x86_64-unknown-freebsd": _support(std = True, host_tools = True),
diff --git a/cargo/settings/settings.bzl b/cargo/settings/settings.bzl
index 5cd0b45..6119613 100644
--- a/cargo/settings/settings.bzl
+++ b/cargo/settings/settings.bzl
@@ -18,6 +18,7 @@ def cargo_manifest_dir_filename_suffixes_to_retain():
         build_setting_default = [
             ".lib",
             ".so",
+            ".a",
         ],
     )
 
