--- src/server/integrations/BUILD.bazel
+++ src/server/integrations/BUILD.bazel
@@ -19,6 +19,7 @@
         "//src/server/integrations/litefs",
         "//src/server/integrations/obsidian",
         "//src/server/integrations/ollama",
+        "//src/server/integrations/mercadopago",
         "//src/server/integrations/powersync",
         "//src/server/integrations/restic",
         "@org_golang_google_protobuf//proto",
